Codemine/Extensions/NSURL+Utilities.swift

```diff
@@ -86,4 +86,34 @@ public extension URL {
         urlComponents.queryItems = urlQueryItems
         return urlComponents.url
     }
+    
+    /**
+     Changes a value for a queryParameter in a given URL
+     - parameters:
+     - url: The `URL` that you want to change a queryParemeter in
+     - withName: The `String` representation of the name of the queryParameter you want to change the value of
+     - toValue: The `String` representation of the new value for the queryParameter
+     - returns: a new `URL` instance with the changed queryParameters or nil if the change failed
+     */
```

Comment:
tip: Xcode can help creating the doc section for a function. Editor > Structure > Add Documentation. Or Command Option /

It saves time and helps create a standardized format for docs.